vulyk/blueprints/gamification/__init__.py

```diff
@@ -20,12 +20,40 @@
 
 from . import listeners
 from .models.events import EventModel
+from .. import VulykModule
 
 __all__ = [
     'gamification'
 ]
 
-gamification = flask.Blueprint('gamification', __name__)
+
+class GamificationModule(VulykModule):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.config["levels"] = {
+            k: 1 if k == 1 else (k - 1) * 25 for k in range(1, 51)
+        }
+
+    def get_level(self, points):
+        """
+        Obtains the level that corresponds to a number of points
+
+        :param points: Number of points
+        :type points: Decimal
+
+        :return: Level
+        :rtype: int
+        """
+
+        for k in sorted(self.config["levels"].keys(), reverse=True):
```

Comment:
>having hardcoded levels above, we may just use floor division by 25.
I'd like to stick to flexible scheme. Current one looks dumb

>swap keys and values to reduce values dereferencing;
That will make data structure less readable, level vs points seems like a more natural way to read it to me

> use OrderedDict;
I can use OrderedDict for sure, but I have no confidence that someone who will redefine that setting in blueprint settings will use it too. Sorting makes the solution bullet-proof (while adding very little overhead). However if you want, I can redefine configure method to apply sorting once and for all.